js/shop.js

```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
+var shopping_tool_url = "https://icommons-tools.tlt.harvard.edu/shopping";    // the url of the shopping tool
+
```

Comment:
More a question for devops I guess. @cmurtaugh/@ekiczek:  wouldn't the shopping_tool_url change  with the tools moving to AWS?